packages/requisitions/src/CustomerRequisition/api/operations.graphql

```diff
@@ -0,0 +1,114 @@
+mutation updateResponseRequisition(
+  $storeId: String!
+  $input: UpdateResponseRequisitionInput!
+) {
+  updateResponseRequisition(input: $input, storeId: $storeId) {
+    ... on RequisitionNode {
+      __typename
+      id
+    }
+  }
+}
+
+fragment ResponseRequisitionLine on RequisitionLineNode {
+  id
+  itemId
```

Comment:
what about adding in 
```
item {
  id
  code
  name
}
```

instead of itemId - to use item name and code in the list? 
either add a custom accessor or flatten the response object..

Reply:
I took the easy route and made an issue https://github.com/openmsupply/openmsupply-client/issues/853

In general I suppose this will be a little easier to develop around when we can add items to a request requisition